tools/debugging/stress_test_transfers.py

```diff
@@ -152,10 +157,25 @@ def subprocess_stopped(result: AsyncResult) -> None:
 
                     # if the subprocess error'ed propagate the error.
                     if result.get() != STATUS_CODE_FOR_SUCCESS:
-                        self.stop.set()
+                        log.error("Raiden died! Bailing out.")
+                        stop.set()
 
                 process.result.rawlink(subprocess_stopped)
 
+            @staticmethod
+            def spawn_under_watch(function: Callable, *args: Any, **kwargs: Any) -> Greenlet:
+                greenlet = gevent.spawn(function, *args, **kwargs)
+
+                # The Event.rawlink is executed inside the Hub thread, which
+                # does validation and *raises on blocking calls*, to go around
+                # this a new greenlet has to be spawned, that in turn will
+                # raise the exceptoin.
+                def spawn_to_kill() -> None:
+                    gevent.spawn(greenlet.throw, gevent.GreenletExit())
+
+                stop.rawlink(lambda _stop: spawn_to_kill())
```

Comment:
Event.rawlink is not in the docs http://www.gevent.org/api/gevent.event.html#gevent.event.Event. Is it safe to use?

Reply:
humm, I don't know if the API is stable. I know this is used a lot in the internals though.

If you want, I can change it to spawn a new greenlet that would call `stop.get()` or the similar, and avoid `rawlink` altogheter.